core/metacling/src/TCling.cxx

```diff
@@ -6922,6 +6922,16 @@ void TCling::CodeComplete(const std::string& line, size_t& cursor,
    fInterpreter->codeComplete(line, cursor, completions);
 }
 
+std::string TCling::Evaluate(const char* code)
```

Comment:
Now, there is `Calc`, `ProcessLine` and `Evaluate`. What's the difference? Why does `Evaluate` only return `std::string`?

Reply:
We're no longer using Calc, instead we're using Evaluate which is essentially doing the same thing as what root prompt does. This way we don't have to call cling::printValue directly, which gives ROOT a chance to load RuntimePrintValue.h.